yt_dlp/extractor/acfun.py

```diff
@@ -0,0 +1,191 @@
+
+from .common import InfoExtractor
+from ..utils import (
+    bug_reports_message,
+    ExtractorError,
+    float_or_none,
+    format_field,
+    int_or_none,
+    traverse_obj,
+    parse_codecs,
+)
+
+
+class AcFunVideoBaseIE(InfoExtractor):
+    def parse_format_and_subtitle(self, video_id, video_info):
+        if 'ksPlayJson' not in video_info:
+            raise ExtractorError(f'Unknown webpage json schema{bug_reports_message()}')
+        playjson = self._parse_json(video_info['ksPlayJson'], video_id)
+
+        fmt_jobj = traverse_obj(playjson, ('adaptationSet', 0, 'representation'))
```

Comment:
No idea what `fmt_jobj` is supposed to mean. Use a better name

Reply:
ok